src/loose/index.js

```diff
@@ -41,9 +41,7 @@ defaultOptions.tabSize = 4
 
 // eslint-disable-next-line camelcase
 export function parse_dammit(input, options) {
-  let p = new LooseParser(input, options)
-  p.next()
-  return p.parseTopLevel()
```

Comment:
`parseTopLevel` doesn't _take_ an argument. To clarify, these are completely different classes. A loose parser _has_ a regular parser, but doesn't inherit from it.